src/main/validateParameters.js

```diff
@@ -81,6 +81,7 @@ function validateParameters(realObj, expectedObj, prohibitNull = true, checkCall
       callingUser: {
         username: PRIMITIVE_TYPES.STRING,
         type: PRIMITIVE_TYPES.STRING,
+        ... expectedObj.callingUser ? expectedObj.callingUser : undefined
```

Comment:
wanna see something cool with how JS works?
what you wrote here is essentially ```(expected.callingUser && expected.callingUser)```
Look, this works as you wanted:
![image](https://github.com/RoiTief/Exam-Management-System/assets/80453712/82a19a6d-1967-4190-9e43-73124f685456)

but writing ```(p && p)``` is ```if (p) then p else p``` so basically ```p``` :
![image](https://github.com/RoiTief/Exam-Management-System/assets/80453712/df94056c-d37a-4fe0-b929-c0a69f4aa780)

Reply:
Ohh cool